extensions/binding/org.eclipse.smarthome.binding.lirc/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="lirc"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- Sample Thing Type -->
+    <thing-type id="remote" listed="false">
+        <label>Remote Control</label>
+        <description>An IR remote control</description>
+
+        <channels>
+            <channel id="button" typeId="button"/>
+        </channels>
+        
+        <config-description>
+            <parameter name="remote" type="text" required="true">
+                <label>Remote name</label>
+                <description>The name of the remote as configured in LIRC</description>
+            </parameter>
+        </config-description>
+        
+    </thing-type>
+
+    <!-- Sample Channel Type -->
+    <channel-type id="button">
+        <item-type>String</item-type>
```

Comment:
The trigger channel makes sense to me. I don't think I've ever seen anything use this type of channel before, so I hadn't even considered it.

This would actually solve a few weird situations I ran into during development, so I'm all for making this change.